migrations/1614771815912-add_encrypted_version.ts

```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class addEncryptedVersion1614771815912 implements MigrationInterface {
+  name = 'addEncryptedVersion1614771815912'
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query('ALTER TABLE `settings` CHANGE `encrypted` `encryption_version` tinyint(1) NOT NULL DEFAULT 0')
+    await queryRunner.query('ALTER TABLE `users` ADD `server_key` varchar(255) NULL')
```

Comment:
If you like explicitness this might be better called `encrypted_server_key` so there is no confusion about it.